src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -54,26 +57,34 @@ public SortCommand(String attribute, boolean isAscending) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         Comparator<Client> comparator;
+        SortCriteria criteria;
+        if (!Objects.equals(attribute, "") && !SortCriteria.isValidEnum(attribute)) {
```

Comment:
Should this be an or instead of an and?

Reply:
Should be && because I attribute being empty is valid (it will sort by default). So I only want to check for when attribute is not empty and not a sortcrtieria.